File: src/main/java/com/uwetrottmann/trakt/v2/entities/BaseEpisode.java
package com.uwetrottmann.trakt.v2.entities;

import org.joda.time.DateTime;

public class BaseEpisode {

    public Integer number;

    /** collection */
    public DateTime collected_at;
    /** watched */
    public Integer plays;
    /** progress */
    public Boolean completed;

}

File: src/main/kotlin/com/redissi/trakt/entities/BaseEpisode.kt
package com.redissi.trakt.entities

import com.squareup.moshi.Json
import java.time.OffsetDateTime

class BaseEpisode(
    val number: Int? = null,
    /** collection  */
    @Json(name = "collected_at")
    val collectedAt: OffsetDateTime? = null,
    /** watched  */
    val plays: Int? = null,
    @Json(name = "last_watched_at")
    val lastWatchedAt: OffsetDateTime? = null,
    /** progress  */
    val completed: Boolean? = null
)